includes/Admin/Settings.php

```diff
@@ -350,6 +350,12 @@ public function get_settings_fields() {
                     'type'  => 'file',
                     'desc'  => __( 'Recommended Logo size ( 270px X 90px ). If no logo is uploaded, site title is shown by default.', 'dokan-lite' ),
                 ],
+                'setup_wizard_message'  => [
+                    'name'      => 'setup_wizard_message',
+                    'label'     => __( 'Vendor Setup Wizard Message', 'dokan-lite' ),
+                    'type'      => 'wpeditor',
```

Comment:
since we are allowing only strong tag via wp_keses, I think wpeditor is overkill here, kindly use  textarea as type with row property set to 3